roles/gateway/tasks/fail2ban.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Write fail2ban defaults conf file
+  template:
+    src: templates/f2b.jail.local.j2
+    dest: /etc/fail2ban/jail.local
+  notify: restart fail2ban
+
+# In case malformed service file was previously shipped.
+# Malformed service files cause fail2ban to not start.
+- name: Clean up ansible-written service conf files
+  shell: for file in $(grep -l "ansible" /etc/fail2ban/jail.d/*); do rm -vf $file; done
```

Comment:
just having "ansible" somewhere in the file seems a little dodgy.  Can we grep for {{ansible_managed}"} instead?